docs/dev/policy_documents/standards-for-tests.md

```diff
@@ -179,33 +180,36 @@ fi
 
 TEST_PKG_IDENT="${1}"
 export TEST_PKG_IDENT
+
 hab pkg install core/bats --binlink
 hab pkg install "${TEST_PKG_IDENT}"
-bats "$(dirname "${0}")/test.bats"
+
+bats "$(dirname "${0}")"/test.bats
 ```
 
 BATS test script `7zip/tests/test.bats`:
 
 ```bash
-TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+expected_version="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+
 @test "7z exe runs" {
-  run hab pkg exec ${TEST_PKG_IDENT} 7z
+  run hab pkg exec $TEST_PKG_IDENT 7z
```

Comment:
@gavindidrichsen Sorry, I missed this in my earlier review.  We should make sure we're quoting variables in our examples.